src/SessionMonitor/index.js

```diff
@@ -80,9 +103,19 @@ export class SessionMonitor {
     // hitting Fence endpoint refreshes token
     return getReduxStore().then((store) => {
       store.dispatch(fetchUser).then((response) => {
+        this.fetchUserAttemptCounter = 0;
         if (response.type === 'UPDATE_POPUP') {
           this.popupShown = true;
         }
+      }).catch(() => {
+        this.fetchUserAttemptCounter += 1;
+        // try again after 5 seconds only retry 9 times
+        // stop after 9 times as this will be triggered again after updateSessionTime
+        if (this.fetchUserAttemptCounter < 10) {
```

Comment:
what are these 9 retries for?

Reply:
good point, don't need the retries as the code automatically retries every few minutes